README.md

```diff
@@ -1431,16 +1431,18 @@
     The usage of a promise would be as below,
 
     ```javascript
-    const promise = new Promise(
-      (resolve) => {
-        setTimeout(() => {
-          resolve("I'm a Promise!");
-        }, 5000);
-      },
-      (reject) => {}
-    );
-
-    promise.then((value) => console.log(value));
+    const promise = new Promise((resolve, reject) => {
+      setTimeout(() => {
+        if (Math.random() > 0.5) { // 50% chance of true
```

Comment:
Thanks for your contribution. 
In my opinion old code should be fine as we will give the same output all the time. 

We can be modified like

- schema
- resolve case
- reject case

Adding more conditions can confuse the beginner.
Thanks
Rahu;